pages/[type]/[id]/version/[version].vue

```diff
@@ -195,29 +195,12 @@
     <div class="version-page__changelog universal-card">
       <h3>Changelog</h3>
       <template v-if="isEditing">
-        <span
-          >This editor supports
-          <a
-            class="text-link"
-            href="https://docs.modrinth.com/docs/tutorials/markdown/"
-            target="_blank"
-            >Markdown formatting</a
-          >. HTML can also be used inside your changelog, not including styles, scripts, and
-          iframes.
+        <span>
+          HTML can also be used inside your changelog, not including styles, scripts, and iframes.
```

Comment:
Not necessary to put this here. It should either be in the same section as the markdown warning in the omorphia component, or it should be omitted entirely.